workflow/rules/samtools.smk

```diff
@@ -0,0 +1,37 @@
+__author__ = "Martin Rippin"
+__copyright__ = "Copyright 2021, Martin Rippin"
+__email__ = "martin.rippin@scilifelab.uu.se"
+__license__ = "GPL3"
+
+
+rule samtools_view:
+    input:
+        bam="alignment/merge_bam/{file}.bam",
+        bai="alignment/merge_bam/{file}.bam.bai",
+        ref=config.get("reference", {}).get("fasta", ""),
+    output:
+        cram=temp("compression/samtools_view/{file}.cram"),
```

Comment:
If you specify and index file as output samtools view will fix that for you as well.